### PR TITLE
M5108 Respect size of logo navigation bar

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [nvt] Code unit/integration/system tested
- [x] User documentation updated
- [nvt] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [will squash ] Clean commits

### CODE REVIEW

File: docs/user_documentation/guide-customize.md [hunk 1]
```diff
@@ -230,3 +230,11 @@ The plugin ID can be found in the URL, the sys_Plugin table, or in the menu mana
 
 After saving, the importwizard plugin will now have a description at the top when the english language is selected.
 This is an admin only feature and allows the admin to use HTML.
+
+# Adding a banner
+If you want to brand your MOLGENIS instance, you could add a banner.
+The width of the banner does not really matter, a higher resolution is better. It will be automatically resize to fit the browser window.
+The maximum height of a banner is **200px**!! Any larger will cause rescaling of your image and it will not look nice.
```
Comment (bartcharbon) [inline docs/user_documentation/guide-customize.md]: is this maximum discussed with @mswertz?

Comment (Mark-de-Haan) [reply docs/user_documentation/guide-customize.md]: Not yet, I will before changing anything

File: molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl [hunk 2]
```diff
@@ -186,14 +211,12 @@
 <#macro topmenu menu plugin_id pluginid_with_query_string>
     <nav class="navbar navbar-default navbar-fixed-top" style="margin-bottom: 10px" role="navigation">
         <div class="container-fluid">
+            <#if app_settings.logoTopHref?has_content>
+            <header id="top-logo-banner">
+                <a href="/"></a><img id="logo-top" src="${app_settings.logoTopHref?html}" alt="" border="0">
```
Comment (bartcharbon) [inline molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: is it on purpose that you close the anchor before the image?

**Comment by bartcharbon** [comment]
Might be useful to have a setting to configure the height of the banner. If the setting is left empty we take the height of the image. For e.g. 

Deb-central has a top logo that is way bigger than we want to show in molgenis.

**Pull request merged**